src/helpers/v2BondDetails.ts

```diff
@@ -1,520 +0,0 @@
-  name: "FXS",
-  bondIconSvg: ["FXS"],
-  pricingFunction: async () => {
-    return getTokenByContract("0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0");
-  },
-  isLP: false,
-  lpUrl: "",
-};
-
-const UstDetails: V2BondDetails = {
-  name: "UST",
-  bondIconSvg: ["UST"],
-  pricingFunction: async () => {
-    return getTokenByContract("0xa693b19d2931d498c5b318df961919bb4aee87a5");
-  },
-  isLP: false,
-  lpUrl: "",
-};
-
-const WbtcDetails: V2BondDetails = {
-  name: "wBTC",
-  bondIconSvg: ["wBTC"],
-  pricingFunction: async () => {
-    return getTokenByContract("0x2260fac5e5542a773aa44fbcfedf7c193bc2c599");
-  },
-  isLP: false,
-  lpUrl: "",
-};
-
-export const UnknownDetails: V2BondDetails = {
-  name: "unknown",
-  bondIconSvg: ["OHM"],
-  pricingFunction: async () => {
-    return 1;
-  },
-  isLP: false,
-  lpUrl: "",
-};
-
-/**
- * DOWNCASE ALL THE ADDRESSES!!! for comparison purposes
- */
-export const singleSidedBondDetails: { [key: number]: { [key: string]: V2BondDetails } } = {
-  [NetworkId.TESTNET_RINKEBY]: {
-    ["0xc0b491dabf3709ee5eb79e603d73289ca6060932"]: OhmDetails,
-    ["0xd7b98050962ec7cc8d11a83446b3217257c754b7"]: OhmDetails,
-    // ["0x14cD61F5d13a4E175F032B252f197B65B9c8E6dc"]: OhmDetails,
-    ["0xb2180448f8945c8cc8ae9809e67d6bd27d8b2f2c"]: DaiDetails,
-    ["0x5ed8bd53b0c3fa3deabd345430b1a3a6a4e8bd7c"]: DaiDetails,
-    ["0xbc9ee0d911739cbc72cd094ada26f56e0c49eeae"]: DaiDetails,
-    ["0x2f7249cb599139e560f0c81c269ab9b04799e453"]: FraxDetails,
-    ["0xc778417e063141139fce010982780140aa0cd5ab"]: EthDetails,
-    // ["0xb2180448f8945c8cc8ae9809e67d6bd27d8b2f2c"]: CvxDetails, // we do not have CVX rinkeby in previous bonds
-    // ["0x80edbf2f58c7b130df962bb485c28188f6b5ed29"]: OhmDaiDetails,
-  },
-  [NetworkId.MAINNET]: {
-    ["0x64aa3364f17a4d01c6f1751fd97c2bd3d7e7f1d5"]: OhmDetails,
-    ["0x04906695d6d12cf5459975d7c3c03356e4ccd460"]: sOhmDetails,
-    ["0x0ab87046fbb341d058f17cbc4c1133f25a20a52f"]: gOhmDetails,
-    ["0x6b175474e89094c44da98b954eedeac495271d0f"]: DaiDetails,
-    ["0x853d955acef822db058eb8505911ed77f175b99e"]: FraxDetails,
-    ["0x5f98805a4e8be255a32880fdec7f6728c6568ba0"]: LusdDetails,
-    ["0xa693b19d2931d498c5b318df961919bb4aee87a5"]: UstDetails,
-    ["0x2260fac5e5542a773aa44fbcfedf7c193bc2c599"]: WbtcDetails,
-    ["0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"]: EthDetails,
-    ["0x4e3fbd56cd56c3e72c1403e103b45db9da5b9d2b"]: CvxDetails,
-    ["0x3432b6a60d23ca0dfca7761b7ab56459d9c964d0"]: FxsDetails,
-    // ["0x69b81152c5a8d35a67b32a4d3772795d96cae4da"]: OhmEthDetails,
-    // ["0x055475920a8c93cffb64d039a8205f7acc7722d3"]: OhmDaiDetails,
-    // ["0xfdf12d1f85b5082877a6e070524f50f6c84faa6b"]: OhmV1LusdDetails,
-  },
-};
-
-/**
- * G-Uni LP will have a variable, ever-changing set of reserve addresses, so parsing code needs to be dynamic
- *
- * BondParser should figure out what to return:
- * 1. logos
- * 2. names
- * 3. pricing
- * 4. isLP
- * 5. lpURL
- */
```

Comment:
This code isn't currently in use, but I'm worried it's in place for something in future to do with balancer and g-uni LP bonds?